impl/src/main/java/org/jboss/cdi/tck/tests/full/interceptors/contract/aroundConstruct/AbstractInterceptor.java

```diff
@@ -14,7 +14,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.jboss.cdi.tck.interceptors.tests.contract.aroundConstruct;
+package org.jboss.cdi.tck.tests.full.interceptors.contract.aroundConstruct;
```

Comment:
Not a request for change, just a question. Some interceptors tests are currently present in `org.jboss.cdi.tck.interceptors.tests`, while others are in `org.jboss.cdi.tck.tests.interceptors`. I _think_ the first ones are for features defined by the Interceptors spec, while the second ones are for features added by CDI. I wonder if we want to keep that distinction?

Reply:
That is a possibility, and would we introduce another test group to distinguish these as not being in lite, or simply label them full?

Reply:
No I think I'd label these tests `CDI_FULL` as usual. I was only commenting on the package structure.

Reply:
Yeam might be good to keep the package structure. It indeed looks like `org.jboss.cdi.tck.interceptors.tests` is basically an interceptors TCK.

Reply:
BTW I am seeing similar issue while looking at tests using portable extensions so I thought it might be sensible to place any interceptor TCK tests that require CDI full to `org.jboss.cdi.tck.interceptors.tests.full` package while keeping any CDI-related tests under the package that you used in this PR.
E.g. if a test needs to be moved, do it in following way:
* [interceptor TCK test] `org.jboss.cdi.tck.interceptors.tests` -> `org.jboss.cdi.tck.interceptors.tests.full`
* [CDI TCK test] `org.jboss.cdi.tck.tests.interceptors` -> `org.jboss.cdi.tck.tests.full.interceptors`